source/agora/node/Ledger.d

```diff
@@ -147,12 +149,28 @@ public class Ledger
 
         return null;
     }
+
+    /***************************************************************************
+
+        Ledger serialization
+
+        Params:
+            block = LedgerOutputStream
+
+    ***************************************************************************/
+
+    public void serialize (LedgerOutputStream stream) @safe
```

Comment:
I would not add a `serialize` method to `Ledger`. Because it is not a data structure that needs serialization, it is a owner of the data (which is likely to know when serialization is needed), thus it does not fall under the same abstraction.

In other word: How often do we want to serialize the full blockchain ?

Reply:
So I deleted it :)